server/helpers/ffmpeg-utils.ts

```diff
@@ -116,28 +116,27 @@ type TranscodeOptions = {
 
 function transcode (options: TranscodeOptions) {
   return new Promise<void>(async (res, rej) => {
+    let fps = await getVideoFileFPS(options.inputPath)
```

Comment:
Why did you move this block? :)

Reply:
Because fps is needed for the preset (just noticed that I forgot to set the parameter correctly).